src/test/java/reposense/parser/GroupConfigParserTest.java

```diff
@@ -0,0 +1,84 @@
+package reposense.parser;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import reposense.model.FileType;
+import reposense.model.GroupConfiguration;
+
+
```

Comment:
Can remove the extra blank line. I think the existing convention is to keep a single blank line between the imports and class body.